net-im/qtox/qtox-1.17.3.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_P="qTox-${PV}"
+inherit cmake xdg
+
+DESCRIPTION="qTox is an instant messaging client using the encrypted p2p Tox protocol"
+HOMEPAGE="https://qtox.github.io/"
+SRC_URI="https://github.com/qTox/qTox/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
```

Comment:
Had to update this to point out to proper release download link, 
https://github.com/qTox/qTox/releases/tag/v1.17.3

Other than that, works perfectly, thanks!